lms/envs/common.py

```diff
@@ -2963,6 +2963,10 @@ def _make_locale_paths(settings):
 # visible. We default this to the legacy permission 'see_exists'.
 COURSE_ABOUT_VISIBILITY_PERMISSION = 'see_exists'
 
+# [COLARAZ_CUSTOM]
+# it will restrict anonymous users from viewing courses on index, courses and course about pages
+SHOW_COURSES_TO_ANONYMOUS_USERS = False
```

Comment:
Please add this config in `production.py` too so that this could be configured through the configurations later.

Reply:
Sorry, couldn't understand your point.

Reply:
This is a new variable. right?
If we want it to be configured through DevOps configurations i.e. through `lms.env.json` file, we have to add this in production.py in the following way:
```
SHOW_COURSES_TO_ANONYMOUS_USERS = ENV_TOKENS.get('SHOW_COURSES_TO_ANONYMOUS_USERS', SHOW_COURSES_TO_ANONYMOUS_USERS)
```

Reply:
As this variable will be used in Site configurations so I don't think there is a need for that but still am going to add it now.

Reply:
SInce we have it in `common.py`, it would be better to have it there too. 
It's always better to have more options :D

Reply:
Not always...

Reply:
done